src/main/plugins/org.dita.pdf2/xsl/fo/topic.xsl

```diff
@@ -53,17 +53,7 @@ See the accompanying LICENSE file for applicable license.
     </xsl:template>
 
     <xsl:template match="*[contains(@class,' topic/topic ')]/*[contains(@class,' topic/title ')]">
-        <xsl:variable name="topicType" as="xs:string">
```

Comment:
This should now be removed, with the change it will run normal processing in every case.

Reply:
@jelovirt - I updated the PR to keep the code but fix the variable reference. Please let me know if this is correct, as I was unable to sensitize this code path to validate the behavior. (We do not use the `pdf2` transformation and I am not familiar with it.)

Reply:
@jelovirt - with the code change to keep the code, do you consider this aspect resolved?